src/components/Card/PCardBase.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div
+    class="p-card"
+    :class="normalizeClass([props.class, card({ variant })])"
+  >
+    <slot />
+  </div>
+</template>
+
+<script setup lang="ts">
+  import { cva, type VariantProps } from 'class-variance-authority'
+  import { normalizeClass, type HTMLAttributes } from 'vue'
+
+  const card = cva(
+    'p-card', {
+      variants: {
+        variant: {
+          ghost: 'p-card-ghost',
+        },
+      },
+      defaultVariants: {
+        variant: null,
+      },
+    },
+  )
+
+  export type CardProps = VariantProps<typeof card>
+
+  const props = defineProps<{
+    class?: HTMLAttributes['class'],
+    variant?: CardProps['variant'],
+  }>()
+</script>
+
+
+<style scoped>
```

Comment:
if we're using BEM classes we shouldn't need to use `scoped`. There is overhead associated with that and it increases specificity. So we should probably remove the scoped attribute from all of these style blocks.